README.md

```diff
@@ -4,18 +4,61 @@ keywords: machine, hetzner, Hetzner Cloud, driver
 title: Hetzner Cloud
 ---
 
-Create Docker machines on [Hetzner Cloud](https://docs.hetzner.cloud/).
+Create [Docker machines](https://github.com/docker/machine) on [Hetzner Cloud](https://docs.hetzner.cloud/).
 
 You need to create a project-sepcific access token under `Access` > `API Tokens` in the project control panel
 and pass that to `docker-machine create` with the `--hetzner-api-token` option.
 
+## Building
+
+Use an up-to-date version of [go](https://golang.org/dl)
+
+### For use
+
+To use the driver, you can download the sources and build it locally:
+
+    # Tell GO where to find sources and where to put binaries 
+    #    see also https://github.com/golang/go/wiki/SettingGOPATH
+    export GOPATH="$(pwd)"
+    export GOBIN=$GOPATH/bin
+    # Build the binary
+    go get github.com/jonasprogrammer/docker-machine-driver-hetzner
+    # Make the binary accessible to docker-machine
+    export PATH="$PATH:$GOBIN"
+    # Make docker-machine output help including hetzner-specific options
+    docker-machine create --driver hetzner
+
+### For development
+
+To compile the local sources, do the following:
+
+    # Check out sources
```

Comment:
To stay compatible with standard go, perhaps recommend setting `$GOPATH` first and then checking the source out into a subdirectory of `/src/` (though not necessarily in the correct package structure)?

Reply:
Hm, I considered that, but didn't do it because the directories that go creates are like `src/github.com/jonasprogrammer/docker-machine-driver-hetzner/` - i.e. more sub-directories than go files. I looked at a small number of other drivers and they all have the go sources in the project root (e.g. https://github.com/docker/docker.github.io/blob/master/machine/AVAILABLE_DRIVER_PLUGINS.md and https://github.com/docker/machine/tree/master/drivers/digitalocean).

My understanding is also that the stuff under $GOPATH is maintained by `go build` and not created directly by developers, e.g. by `git clone`/`git pull`. But thanks to your comment I now figured out that `go get` clones the git repo itself and puts it into a global structure typically at `~/go/`. I'll adjust the README accordingly.